src/Components/App/App.js

```diff
@@ -28,8 +28,8 @@ function App() {
 
   return (
     <>
-      <button onClick={() => handleClick("add")}>ADD</button>
```

Comment:
thanks for commenting these out so we can use them as templates or notes for future use